### ISSUE TITLE
SignatureDoesNotMatch on POST to AWS SQS API with JSON MessageBody

### ISSUE THREAD

**Issue by peyerc** [a85/POSTMan-Chrome-Extension#5996]
**Describe the bug**
I tried to use the AWS SQS API with Postman and was not able to successfully execute a POST with a JSON in the MessageBody. When I do the same request as a GET, it works as expected. The POST works with a simple word as MessageBody though.

**To Reproduce**
1. In AWS I've setup a SQS FIFO Queue and combined it with a user with enough permissions to send messages.
2. In Postman I've configured the POST request with an "AWS Signature" authorization and a "x-www-form-urlencoded" body.
3. The body contains the following fields and values:
* Action:SendMessage
* MessageBody:{"key": "value"}
* Version:2012-11-05
* MessageGroupId:12345
* MessageDeduplicationId:12345
4. The error I get is the following
```
<?xml version="1.0"?>
<ErrorResponse xmlns="http://queue.amazonaws.com/doc/2012-11-05/">
    <Error>
        <Type>Sender</Type>
        <Code>SignatureDoesNotMatch</Code>
        <Message>The request signature we calculated does not match the signature you provided. Check your AWS Secret Access Key and signing method. Consult the service documentation for details.

The Canonical String for this request should have been
'POST
/533692402829/test_drm_callback.fifo

cache-control:no-cache
content-length:116
content-type:application/x-www-form-urlencoded
host:sqs.eu-west-1.amazonaws.com
postman-token:54580eab-167a-4518-9366-15ce2abd4634
x-amz-date:20190228T073548Z

cache-control;content-length;content-type;host;postman-token;x-amz-date
b9def332e5a33a38d10070439410952895c3127aca303cd8e298a1f7c92183eb'

The String-to-Sign should have been
'AWS4-HMAC-SHA256
20190228T073548Z
20190228/eu-west-1/sqs/aws4_request
d94df532b2396cdb1e0f85cc2ca9d838cde3c3e3b1d73cd77f44e8c455fb9c27'
</Message>
        <Detail/>
    </Error>
    <RequestId>b308dd1c-39e1-5c80-9b2b-89eda874026e</RequestId>
</ErrorResponse>
```

**Expected behavior**
I can successfully POST a Message with an JSON MessageBody to the SQS Queue.

**App information (please complete the following information):**
 - Native App
 - Postman Version 6.7.4
 - OS: macOS Mojave 10.14.3

**Comment by codenirvana** [comment]
We've released a fix for this on our Canary (`version: 7.1.0-canary01`) channel https://www.getpostman.com/canary

Please try it out in the Canary version and let us know if you continue to face the issue.
Will update the thread once we release the fix on our stable channel.

**Comment by codenirvana** [comment]
This has been fixed in the latest Postman app.
